src/lib/ui/DeviceDetails/DeviceModel.tsx

```diff
@@ -0,0 +1,25 @@
+import type { LockDevice } from 'seamapi'
+
+import { getLockModel } from 'lib/index.js'
+
+interface ModelStatusProps {
+  device: LockDevice
+}
+
+export function DeviceModel({ device }: ModelStatusProps): JSX.Element | null {
+  const model = getLockModel(device)
+  if (!model) {
```

Comment:
The lint rule was unintentionally disabled, I mean to correct that.
```suggestion
  if (model == null) {
```

Reply:
Ok, this is going to take some getting used to, very used to typing `===`. I get the readability of it though, so will try to remember to `== null` instead

Reply:
This is a JS idiosyncrasy that I hate but have accepted as reality. `x === null` is too strict, `x === null || x === undefined` is too verbose, `!x` works for both, but also `!0 === true` so it's too broad. Same thing with `!==` but it's worse, since you end up with `!!x` sometimes. But, `x == null` and `x != null` check for what you need and nothing else, even though it breaks the `===` "rule"